libmscore/timesig.cpp

```diff
@@ -299,6 +299,27 @@ void TimeSig::layout()
             ds.clear();
             }
       else {
+            // "other" timesigs are placed at the middle of the staff too.
+            const std::vector<SymId> prolatioIDList = {
+                  SymId::mensuralProlation1, SymId::mensuralProlation2, SymId::mensuralProlation3,
+                  SymId::mensuralProlation4, SymId::mensuralProlation5, SymId::mensuralProlation6,
```

Comment:
SymId::mensuralProlation6 is here, but missing in other places (like libmscore/utils.cpp and mscore/timsigproerties.cpp), why?

Reply:
OK, probably Prolation6 and Prolation9 are four four and alla breve...

Reply:
Same for `SymId::mensuralProlation`

Reply:
Thanks, the change version looks better to me, less code duplication, less inconsistency

Reply:
OK...
Do you have any idea why switching from 4/4 or alla breve to other won't work?

Reply:
No, sorry